src/Libraries/Microsoft.Extensions.AI/Embeddings/NormalizationEmbeddingGenerator.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#if NET // requires generic math interfaces
```

Comment:
Should we consider authoring specialized implementations `double` and `float` in ns2.0?

Reply:
I think for now it's reasonable to say that this is only available on core.